local-volume/provisioner/pkg/common/common.go

```diff
@@ -57,6 +64,8 @@ type MountConfig struct {
 	HostDir string `json:"hostDir"`
 	// The mount point of the hostpath volume
 	MountDir string `json:"mountDir"`
+	// The volumeType "block" or "file". If nil, the type defaults to "file"
+	VolumeType string `json:"volumeType"`
```

Comment:
The discussion last week was to not have fstype in the PVC, and that local storage wouldn't support the file-on-block case for now.  But I think volume type still has to be specified, so you can only bind a PVC with volume type matching on the PV.